de.dlr.sc.virsat.model.extension.mechanical.test/src/de/dlr/sc/virsat/model/extension/mechanical/catia/util/CatiaHelperTest.java

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.model.extension.mechanical.catia.util;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.github.cliftonlabs.json_simple.JsonArray;
+import com.github.cliftonlabs.json_simple.JsonObject;
+
+import de.dlr.sc.virsat.model.extension.mechanical.catia.CatiaProperties;
+
+/**
+ * Test class for the CATIA helper class
+ *
+ */
+public class CatiaHelperTest {
+
+	public static final String ELEMENT_1_UUID = UUID.randomUUID().toString();
+	public static final String ELEMENT_2_UUID = UUID.randomUUID().toString();
+	public static final String ELEMENT_3_UUID = UUID.randomUUID().toString();
+
+	public static final int NUMBER_OBJECTS = 4; // Three elements with UUID and the root product element
+
+	@Before
+	public void setUp() {
+
+	}
+
+	@Test
+	public void testGetListOfAllJSONElements() {
+
+		JsonObject rootObject = createJsonObjectWithProductandConfiguration();

```

Comment:
`createJsonObjectWithProductandConfiguration()` should be `createJsonObjectWithProductAndConfiguration()`